ubuntu20.04/val-verde-platform-sdk-swift

```diff
@@ -94,7 +94,8 @@ COMPONENTS="\
 source ${BUILD_PACKAGE_PREFIX}/bin/${PACKAGE_BASE_NAME}-platform-sdk-rpath-fixup
 
 # Build the package
-source ${BUILD_PACKAGE_PREFIX}/bin/${PACKAGE_BASE_NAME}-platform-sdk-package-build
+DEPENDENCIES="${PACKAGE_BASE_NAME}-llvm-project-${HOST_OS}${HOST_OS_API_LEVEL}-${HOST_ARCH}"
+source ${BUILD_PACKAGE_PREFIX}/bin/${PACKAGE_BASE_NAME}-platform-sdk-package-build "${DEPENDENCIES}"
```

Comment:
Same comment as above about escaping DEPENDENCIES with the source line.

Reply:
removed this file to address deb generation as a separate PR